src/models/user.js

```diff
@@ -36,6 +38,17 @@ class User extends db.Model {
   roles() {
     return this.belongsToMany(Role, 'roles_to_users', 'user_id', 'role_id');
   }
+
+  /**
+   * Serialize user data.
+   */
+  serialize() {
```

Comment:
I feel this is very ambigious, we might need to create other filters later that could be inside this. While calling this function, it won't be clearer what serialization does it do. Can we be more clearer by renaming into some thing like `filterSensitiveData()` instead.